### PR TITLE
Fixes bread dog butchering producing error bread and adds some warnings to bread that shouldn't appear in-game.

### LINKED ISSUE #73765: butchering bread dog creates errors
Reporting client version: 514.1589

## Round ID:
[201132](https://scrubby.melonmesa.com/round/201132)

## Reproduction:
butchering a bread dog creates error items named food
not sure if it happens to cat's as well
Capture

### PR DESCRIPTION
**Pull request by Helg2**
## About The Pull Request
Fixes https://github.com/tgstation/tgstation/issues/73765
Also adds some warnings and description to bread/breadslice so you can more easily notice that something is wrong.
Not much to discuss i think.
## Why It's Good For The Game
You can get normal bread from butchering bread dog. And potentially less issues with bread in future.
## Changelog
:cl:
fix: fixed butchering bread dog spawning error bread.
/:cl:

### CODE REVIEW

**Comment by Helg2** [comment]
Also, thanks to @Sealed101, he kinda described the whole issue.

File: code/game/objects/items/food/bread.dm [hunk 1]
```diff
@@ -1,5 +1,7 @@
 
-/obj/item/food/bread
+/obj/item/food/bread //If you'll try to spawn it will occure in error sprite. Shouldn't do this.
```
Comment (Jacquerel) [inline code/game/objects/items/food/bread.dm:2]:
```suggestion
/// Abstract parent object for bread items. Should not be made obtainable in game.
/obj/item/food/bread
```
Just a slight change for grammar/clarity.

File: code/game/objects/items/food/bread.dm [hunk 2]
```diff
@@ -20,7 +22,9 @@
 		AddElement(/datum/element/processable, TOOL_KNIFE, slice_type, yield, 3 SECONDS, table_required = TRUE, screentip_verb = "Slice")
 		AddElement(/datum/element/processable, TOOL_SAW, slice_type, yield, 4 SECONDS, table_required = TRUE, screentip_verb = "Slice")
 
-/obj/item/food/breadslice
+/obj/item/food/breadslice //If you'll try to spawn it will occure in error sprite. Shouldn't do this.
```
Comment (Jacquerel) [inline code/game/objects/items/food/bread.dm:25]:
```suggestion
/// Abstract parent object for sliced bread items. Should not be made obtainable in game.
/obj/item/food/breadslice
```

File: code/game/objects/items/food/bread.dm (hunk 1 above)
Comment (Helg2) [reply code/game/objects/items/food/bread.dm]: 👍

**Pull request merged**